packages/prosemirror-lwdita-demo/src/example.ts

```diff
@@ -70,6 +70,8 @@ jsonDocLoader.then(jsonDoc => {
   a.href = '/';
   a.addEventListener('click', () => location.reload());
   const el: HTMLDivElement | null = document.querySelector("#editor");
+  localStorage.clear();
```

Comment:
This does not fall within the PR description, it's better to submit this as a hotfix instead.

Reply:
This fix was required for testing the conversion and therefore needed to be available in this PR.